src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Performance/BasicPreferLengthCountIsEmptyOverAnyFixer.vb

```diff
@@ -0,0 +1,134 @@
+Imports System.Composition
```

Comment:
```suggestion
' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.

Imports System.Composition
```

Reply:
@mavasani @jmarolf when VB code missing the copyright header the VB build fails without any error/warning info locally. On CI the build never stops, not sure if it is known issue or it was expected

Reply:
@buyaa-n Consider opening an issue to dotnet/roslyn with a commit hash that reproduces the issue

Reply:
I am not sure it is roslyn issue, so [created an issue in this repo](https://github.com/dotnet/roslyn-analyzers/issues/6417), we can transfer if needed

Reply:
For some reason, I can't reproduce the same behavior.

though I reproduce a different seemingly buggy behavior. For me, locally, removing the file header causes the build to pass completely, only in the IDE there is a warning.

![image](https://user-images.githubusercontent.com/31348972/211090357-87abcf92-22e6-4551-9de3-0c49332f4883.png)